2020-07-27 16:35:31.826585 (MainThread): Done resetting log handlers.
2020-07-27 16:35:31.826624 (MainThread): Received response from rpc task.
2020-07-27 16:35:31.826649 (MainThread): Updating status to Paused
2020-07-27 16:40:22.329301 (MainThread): Caught sighup
2020-07-27 16:40:23.125597 (MainThread): Updating status to Ready
2020-07-27 16:40:23.136516 (MainThread): Kicking off rpc task.
2020-07-27 16:40:23.136591 (MainThread): dbt client called with ['--partial-parse', 'rpc', '--profile', 'user']
2020-07-27 16:40:23.139648 (MainThread): Setting up log handlers...
2020-07-27 16:40:23.154282 (MainThread): Running with dbt=0.17.1
2020-07-27 16:40:23.356299 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.rpc.server.RPCServerTask'>, debug=False, exclude=None, host='0.0.0.0', log_cache_events=False, log_format='default', models=None, partial_parse=True, port=8580, profile='user', profiles_dir='/usr/src/develop/.dbt', project_dir=None, record_timing_info=None, rpc_method=None, single_threaded=False, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, vars='{}', version_check=True, warn_error=False, which='rpc', write_json=True)
2020-07-27 16:40:23.387274 (MainThread): Tracking: tracking
2020-07-27 16:40:23.387502 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089d0f3be0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089d0f3a58>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089fef95c0>]}
2020-07-27 16:40:23.387974 (MainThread): Serving RPC server at 0.0.0.0:8580, pid=13
2020-07-27 16:40:23.388696 (MainThread): Supported methods: ['cli_args', 'compile', 'compile_sql', 'deps', 'docs.generate', 'gc', 'get-manifest', 'kill', 'poll', 'ps', 'run', 'run-operation', 'run_sql', 'seed', 'snapshot', 'snapshot-freshness', 'status', 'test']
2020-07-27 16:40:23.389274 (MainThread): Send requests to http://localhost:8580/jsonrpc
2020-07-27 16:40:23.392006 (Thread-1): Parsing macros/core.sql
2020-07-27 16:40:23.396880 (Thread-1): Parsing macros/materializations/helpers.sql
2020-07-27 16:40:23.405943 (Thread-1): Parsing macros/materializations/common/merge.sql
2020-07-27 16:40:23.422035 (Thread-1): Parsing macros/materializations/table/table.sql
2020-07-27 16:40:23.432629 (Thread-1): Parsing macros/materializations/view/create_or_replace_view.sql
2020-07-27 16:40:23.440084 (Thread-1): Parsing macros/materializations/view/view.sql
2020-07-27 16:40:23.447231 (Thread-1): Parsing macros/materializations/snapshot/snapshot.sql
2020-07-27 16:40:23.477026 (Thread-1): Parsing macros/materializations/snapshot/strategies.sql
2020-07-27 16:40:23.495545 (Thread-1): Parsing macros/materializations/snapshot/snapshot_merge.sql
2020-07-27 16:40:23.497445 (Thread-1): Parsing macros/materializations/seed/seed.sql
2020-07-27 16:40:23.521025 (Thread-1): Parsing macros/materializations/incremental/incremental.sql
2020-07-27 16:40:23.531818 (Thread-1): Parsing macros/materializations/incremental/helpers.sql
2020-07-27 16:40:23.535024 (Thread-1): Parsing macros/etc/query.sql
2020-07-27 16:40:23.536792 (Thread-1): Parsing macros/etc/get_custom_schema.sql
2020-07-27 16:40:23.540306 (Thread-1): Parsing macros/etc/get_custom_alias.sql
2020-07-27 16:40:23.541619 (Thread-1): Parsing macros/etc/datetime.sql
2020-07-27 16:40:23.550783 (Thread-1): Parsing macros/etc/get_custom_database.sql
2020-07-27 16:40:23.552515 (Thread-1): Parsing macros/etc/is_incremental.sql
2020-07-27 16:40:23.554270 (Thread-1): Parsing macros/adapters/common.sql
2020-07-27 16:40:23.616644 (Thread-1): Parsing macros/schema_tests/relationships.sql
2020-07-27 16:40:23.617976 (Thread-1): Parsing macros/schema_tests/unique.sql
2020-07-27 16:40:23.619206 (Thread-1): Parsing macros/schema_tests/accepted_values.sql
2020-07-27 16:40:23.621257 (Thread-1): Parsing macros/schema_tests/not_null.sql
2020-07-27 16:40:23.622772 (Thread-1): Parsing macros/adapters.sql
2020-07-27 16:40:23.646000 (Thread-1): Parsing macros/catalog.sql
2020-07-27 16:40:23.648063 (Thread-1): Parsing macros/materializations/incremental.sql
2020-07-27 16:40:23.657752 (Thread-1): Parsing macros/materializations/merge.sql
2020-07-27 16:40:23.660066 (Thread-1): Parsing macros/materializations/table.sql
2020-07-27 16:40:23.664090 (Thread-1): Parsing macros/materializations/view.sql
2020-07-27 16:40:23.693896 (Thread-1): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - my_new_project

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0
2020-07-27 16:40:23.857857 (Thread-2): handling status request
2020-07-27 16:40:23.858544 (Thread-2): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c8aa198>]}
2020-07-27 16:40:23.860437 (Thread-2): sending response (<Response 183 bytes [200 OK]>) to 10.0.5.185
2020-07-27 16:40:23.965790 (Thread-1): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:40:25.114098 (Thread-3): handling status request
2020-07-27 16:40:25.114760 (Thread-3): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c4c2d68>]}
2020-07-27 16:40:25.119858 (Thread-3): sending response (<Response 9151 bytes [200 OK]>) to 10.0.38.156
2020-07-27 16:40:58.637889 (Thread-4): handling status request
2020-07-27 16:40:58.638367 (Thread-4): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c75e630>]}
2020-07-27 16:40:58.643795 (Thread-4): sending response (<Response 9151 bytes [200 OK]>) to 10.0.44.27
2020-07-27 16:41:03.658184 (Thread-5): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:41:03.672059 (Thread-6): handling status request
2020-07-27 16:41:03.672500 (Thread-6): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c4bd0f0>]}
2020-07-27 16:41:03.673357 (Thread-6): sending response (<Response 604 bytes [200 OK]>) to 10.0.35.215
2020-07-27 16:41:20.606966 (Thread-7): handling status request
2020-07-27 16:41:20.607447 (Thread-7): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c8a8a20>]}
2020-07-27 16:41:20.608372 (Thread-7): sending response (<Response 604 bytes [200 OK]>) to 10.0.23.0
2020-07-27 16:41:33.284532 (Thread-8): Got an acceptable cached parse result
2020-07-27 16:41:33.480863 (Thread-8): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:41:33.481507 (Thread-9): handling status request
2020-07-27 16:41:33.482383 (Thread-9): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c4da4e0>]}
2020-07-27 16:41:33.483424 (Thread-9): sending response (<Response 871 bytes [200 OK]>) to 10.0.38.156
2020-07-27 16:41:42.868585 (Thread-10): handling status request
2020-07-27 16:41:42.870398 (Thread-10): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c48f748>]}
2020-07-27 16:41:42.871455 (Thread-10): sending response (<Response 871 bytes [200 OK]>) to 10.0.4.143
2020-07-27 16:41:47.905531 (Thread-11): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:41:47.906165 (Thread-12): handling status request
2020-07-27 16:41:47.907013 (Thread-12): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c0aae10>]}
2020-07-27 16:41:47.907899 (Thread-12): sending response (<Response 605 bytes [200 OK]>) to 10.0.22.37
2020-07-27 16:41:58.252050 (Thread-13): handling status request
2020-07-27 16:41:58.253739 (Thread-13): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c0daf60>]}
2020-07-27 16:41:58.254686 (Thread-13): sending response (<Response 605 bytes [200 OK]>) to 10.0.23.0
2020-07-27 16:42:03.460553 (Thread-14): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:42:03.566875 (Thread-15): handling status request
2020-07-27 16:42:03.567396 (Thread-15): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c045198>]}
2020-07-27 16:42:03.568278 (Thread-15): sending response (<Response 605 bytes [200 OK]>) to 10.0.22.228
2020-07-27 16:45:36.984610 (Thread-16): handling status request
2020-07-27 16:45:36.985108 (Thread-16): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c045e48>]}
2020-07-27 16:45:36.985979 (Thread-16): sending response (<Response 605 bytes [200 OK]>) to 10.0.0.174
2020-07-27 16:45:37.085480 (Thread-17): handling ps request
2020-07-27 16:45:37.085956 (Thread-17): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c045e48>]}
2020-07-27 16:45:37.087898 (Thread-17): sending response (<Response 103 bytes [200 OK]>) to 10.0.23.0
2020-07-27 16:46:08.764026 (Thread-18): handling status request
2020-07-27 16:46:08.765768 (Thread-18): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c0458d0>]}
2020-07-27 16:46:08.766731 (Thread-18): sending response (<Response 605 bytes [200 OK]>) to 10.0.4.143
2020-07-27 16:46:13.854258 (Thread-19): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:46:14.029003 (Thread-20): handling status request
2020-07-27 16:46:14.029484 (Thread-20): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f08977eed30>]}
2020-07-27 16:46:14.030381 (Thread-20): sending response (<Response 605 bytes [200 OK]>) to 10.0.7.30
2020-07-27 16:46:27.245760 (Thread-21): handling status request
2020-07-27 16:46:27.246235 (Thread-21): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c6fc0f0>]}
2020-07-27 16:46:27.247167 (Thread-21): sending response (<Response 605 bytes [200 OK]>) to 10.0.3.235
2020-07-27 16:48:21.001241 (Thread-22): handling status request
2020-07-27 16:48:21.001732 (Thread-22): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c6fc0f0>]}
2020-07-27 16:48:21.002832 (Thread-22): sending response (<Response 605 bytes [200 OK]>) to 10.0.23.0
2020-07-27 16:48:38.983458 (Thread-23): handling status request
2020-07-27 16:48:38.983924 (Thread-23): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c6fc0f0>]}
2020-07-27 16:48:38.984803 (Thread-23): sending response (<Response 605 bytes [200 OK]>) to 10.0.22.37
2020-07-27 16:49:16.505673 (Thread-24): Got an acceptable cached parse result
2020-07-27 16:49:16.685045 (Thread-24): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:49:17.004722 (Thread-25): handling status request
2020-07-27 16:49:17.005233 (Thread-25): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c049f60>]}
2020-07-27 16:49:17.006486 (Thread-25): sending response (<Response 873 bytes [200 OK]>) to 10.0.3.235
2020-07-27 16:49:26.275513 (Thread-26): Got an acceptable cached parse result
2020-07-27 16:49:26.443889 (Thread-26): WARNING: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_new_project.example

2020-07-27 16:49:26.445089 (Thread-27): handling status request
2020-07-27 16:49:26.445588 (Thread-27): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c3ed0f0>]}
2020-07-27 16:49:26.446669 (Thread-27): sending response (<Response 873 bytes [200 OK]>) to 10.0.5.185
2020-07-27 16:49:35.101301 (Thread-28): handling status request
2020-07-27 16:49:35.102974 (Thread-28): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': 'd21d4f0e-a620-486c-827e-9cfb405bca6d', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c79cf98>]}
2020-07-27 16:49:35.103977 (Thread-28): sending response (<Response 873 bytes [200 OK]>) to 10.0.8.48
2020-07-27 16:49:36.903835 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f08b01a9b00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c88b630>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f089c3cba58>]}
2020-07-27 16:49:36.904146 (MainThread): Flushing usage events
2020-07-27 16:49:37.052687 (MainThread): Runtime Error
  Syntax error near line 4
  ------------------------------
  1  | packages:
  2  |   - package: fivetran/salesforce
  3  | <<<<<<< HEAD
  4  |     version: 0.1.0
  5  | =======
  6  |     version: 0.1.0
  7  | >>>>>>> ffe9a6d9fba872156b01f711909f6e015a24b252
  
  Raw Error:
  ------------------------------
  while scanning a simple key
    in "<unicode string>", line 3, column 1:
      <<<<<<< HEAD
      ^
  could not find expected ':'
    in "<unicode string>", line 4, column 12:
          version: 0.1.0
                 ^
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/dbt/clients/yaml_helper.py", line 62, in load_yaml_text
    return safe_load(contents)
  File "/usr/local/lib/python3.6/dist-packages/dbt/clients/yaml_helper.py", line 57, in safe_load
    return yaml.load(contents, Loader=YamlLoader)
  File "/usr/local/lib/python3.6/dist-packages/yaml/__init__.py", line 72, in load
    return loader.get_single_data()
  File "/usr/local/lib/python3.6/dist-packages/yaml/constructor.py", line 35, in get_single_data
    node = self.get_single_node()
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/local/lib/python3.6/dist-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/usr/local/lib/python3.6/dist-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/local/lib/python3.6/dist-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/usr/local/lib/python3.6/dist-packages/yaml/scanner.py", line 115, in check_token
    while self.need_more_tokens():
  File "/usr/local/lib/python3.6/dist-packages/yaml/scanner.py", line 149, in need_more_tokens
    self.stale_possible_simple_keys()
  File "/usr/local/lib/python3.6/dist-packages/yaml/scanner.py", line 289, in stale_possible_simple_keys
    "could not find expected ':'", self.get_mark())
yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 3, column 1:
    <<<<<<< HEAD
    ^
could not find expected ':'
  in "<unicode string>", line 4, column 12:
        version: 0.1.0
               ^

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/app/sinter/clients/dbt.py", line 755, in call
    command + extra_args
  File "/usr/local/lib/python3.6/dist-packages/dbt/main.py", line 159, in handle_and_check
    task, res = run_from_args(parsed)
  File "/usr/local/lib/python3.6/dist-packages/dbt/main.py", line 212, in run_from_args
    results = task.run()
  File "/usr/local/lib/python3.6/dist-packages/dbt/task/rpc/server.py", line 147, in run
    self.run_forever()
  File "/usr/local/lib/python3.6/dist-packages/dbt/task/rpc/server.py", line 142, in run_forever
    threaded=not self.task_manager.single_threaded(),
  File "/usr/local/lib/python3.6/dist-packages/werkzeug/serving.py", line 1012, in run_simple
    inner()
  File "/usr/local/lib/python3.6/dist-packages/werkzeug/serving.py", line 969, in inner
    srv.serve_forever()
  File "/usr/local/lib/python3.6/dist-packages/werkzeug/serving.py", line 737, in serve_forever
    HTTPServer.serve_forever(self)
  File "/usr/lib/python3.6/socketserver.py", line 236, in serve_forever
    ready = selector.select(poll_interval)
  File "/usr/lib/python3.6/selectors.py", line 376, in select
    fd_event_list = self._poll.poll(timeout)
  File "/usr/local/lib/python3.6/dist-packages/dbt/task/rpc/server.py", line 102, in _sighup_handler
    self.task_manager.reload_config()
  File "/usr/local/lib/python3.6/dist-packages/dbt/rpc/task_manager.py", line 128, in reload_config
    config = self.config.from_args(self.args)
  File "/usr/local/lib/python3.6/dist-packages/dbt/config/runtime.py", line 210, in from_args
    project, profile = cls.collect_parts(args)
  File "/usr/local/lib/python3.6/dist-packages/dbt/config/runtime.py", line 196, in collect_parts
    project = partial.render(project_renderer)
  File "/usr/local/lib/python3.6/dist-packages/dbt/config/project.py", line 213, in render
    packages_dict = package_data_from_root(self.project_root)
  File "/usr/local/lib/python3.6/dist-packages/dbt/config/project.py", line 101, in package_data_from_root
    packages_dict = _load_yaml(package_filepath)
  File "/usr/local/lib/python3.6/dist-packages/dbt/config/project.py", line 92, in _load_yaml
    return load_yaml_text(contents)
  File "/usr/local/lib/python3.6/dist-packages/dbt/clients/yaml_helper.py", line 69, in load_yaml_text
    raise dbt.exceptions.ValidationException(error)
dbt.exceptions.ValidationException: Runtime Error
  Syntax error near line 4
  ------------------------------
  1  | packages:
  2  |   - package: fivetran/salesforce
  3  | <<<<<<< HEAD
  4  |     version: 0.1.0
  5  | =======
  6  |     version: 0.1.0
  7  | >>>>>>> ffe9a6d9fba872156b01f711909f6e015a24b252
  
  Raw Error:
  ------------------------------
  while scanning a simple key
    in "<unicode string>", line 3, column 1:
      <<<<<<< HEAD
      ^
  could not find expected ':'
    in "<unicode string>", line 4, column 12:
          version: 0.1.0
                 ^
2020-07-27 16:49:37.055553 (MainThread): Runtime Error
  Syntax error near line 4
  ------------------------------
  1  | packages:
  2  |   - package: fivetran/salesforce
  3  | <<<<<<< HEAD
  4  |     version: 0.1.0
  5  | =======
  6  |     version: 0.1.0
  7  | >>>>>>> ffe9a6d9fba872156b01f711909f6e015a24b252
  
  Raw Error:
  ------------------------------
  while scanning a simple key
    in "<unicode string>", line 3, column 1:
      <<<<<<< HEAD
      ^
  could not find expected ':'
    in "<unicode string>", line 4, column 12:
          version: 0.1.0
                 ^
